### DISCUSSION TITLE
Redis Persistence

### DISCUSSION THREAD

**Discussion by bondhan** [orkes-io/orkes-conductor-community#42] [Q&A]
Hi, 

In the readme it is mentioned that Redis is used to store running workflow. That being said, what is the best setup for production to avoid any lost during workflow execution. 

We are exploring the cluster-Redis. Do we need to setup durable Redis and append only? Any guidance is much appreciated.

BR

**Comment by v1r3n** [comment]
I would suggest configuring redis with AOF or RDB persistence.
